clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -60,34 +61,29 @@ fun <T : Any> EntityPostResponseBody<Any>.toList(cls: KClass<T>): List<T> =
     }
 
 /**
- * Get a FossID project
- * @param user the user querying the API
- * @param apiKey the key to query the API
- * @param projectCode the code of the project
+ * Get a FossID project [projectCode]
```

Comment:
"Get the project for the given [projectCode]." (Note the dot at the end of the sentence.)

Reply:
Similar for the other function docs.